### PR TITLE
Add `pkg` to file-table-export

### PR DESCRIPTION
**Pull request by jez**
### Motivation

I think it will come in useful to have the package information in
Sorbet's file table export. The export is already pretty small, so it's
not that much extra data to export.

### Test plan

See included automated tests.

### CODE REVIEW

**Comment by jez** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6941** <a href="https://app.graphite.dev/github/pr/sorbet/sorbet/6941" target="_blank">Graphite</a> 
    * **PR #6919** <a href="https://app.graphite.dev/github/pr/sorbet/sorbet/6919" target="_blank">Graphite</a> 
      * **PR #6945** <a href="https://app.graphite.dev/github/pr/sorbet/sorbet/6945" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sorbet/sorbet/6945?utm_source=stack-comment).

**Comment by jez** [comment]
We have a policy of testing changes to Sorbet against Stripe's codebase before
merging them. I've kicked off a test run for the current PR.  When the build
finishes, I'll share with you whether or how it failed. Thanks!

Stripe employees can see the build results here:

→ https://go/builds/bui_NiNSoAB0JAUx68
→ https://go/builds/bui_NiNSIoDiHrIe3b
→ https://go/builds/bui_NiNSyk0QjYOg7I

File: test/cli/package-file-table/test.out [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                                         ^
+Errors: 2
+{
+ "files": [
+  {
+   "path": "./bar/__package.rb",
+   "sigil": "Strict",
+   "strict": "Strict",
+   "min_error_level": "Max",
+   "enclosing_package": "Project::Bar"
```
Comment (neilparikh) [inline test/cli/package-file-table/test.out:30]: These should all be `pkg`.

**Review by neilparikh — APPROVED** [review]

**Review by froydnj — APPROVED** [review]
I've always wanted a graph of number of packages over time.

**Comment by jez** [comment]
We have a policy of testing changes to Sorbet against Stripe's codebase before
merging them. I've kicked off a test run for the current PR.  When the build
finishes, I'll share with you whether or how it failed. Thanks!

Stripe employees can see the build results here:

→ https://go/builds/bui_NjT8gi3N6cyyaZ
→ https://go/builds/bui_NjT8nY31hYFcld
→ https://go/builds/bui_NjT8kfJsrlaUsK

**Pull request merged**